tests/api.rs

```diff
@@ -157,31 +160,20 @@ async fn hello_composedb() {
     }
 }
 
-#[tokio::test]
-#[traced_test]
-async fn hello_ceramic() {
```

Comment:
Instead of removing this code can we just ignore the test? https://doc.rust-lang.org/book/ch11-02-running-tests.html#ignoring-some-tests-unless-specifically-requested